editor.planx.uk/src/lib/featureFlags.ts

```diff
@@ -2,6 +2,7 @@
 const AVAILABLE_FEATURE_FLAGS = [
   "DISABLE_SAVE_AND_RETURN",
   "ALT_THEME",
+  "NAVIGATION_BAR",
```

Comment:
nit: could we name this a bit more generally, eg `SECTIONS_NAVIGATION`, since it will also likely control other UI-bits like showing sections on the Review page or not?